src/bridge/types.js

```diff
@@ -106,7 +106,11 @@ export interface AccountBridge<Transaction> {
   // - if promise is successful with null, all is fine
   // - if promise is successful with an error object, it's a warning to display
   // - if promise is unsuccessful, it's an error
-  checkValidRecipient(currency: Currency, recipient: string): Promise<?Error>;
+  checkValidRecipient(
+    currency: Currency,
+    recipient: string,
+    source?: string,
+  ): Promise<?Error>;
```

Comment:
For the format, I would prefer instead we just do this:

`checkValidRecipient(account: Account, recipient: string)`

later we could have more conditions and better just send the whole account object.

this will means changing a bit all the existing bridge but it's just about replacing `currency` param by `{currency}` since it's part of Account.